Suggested Improvements

rly0nheart opened the pull request:
- [x] Now takes arguments, as requested in issue #2 (-i/--ip takes an ip address and -p/--port takes a port number) 
- [x] Payload will have to be built as follows; 'python c2c.py build -i {ip} -p {port}'
- [x]  To listen for connections, users will have to run it as follows: 'python c2c.py listen -i {ip} -p {port}'
- [x] Separated the banner from the main file; assets/banner
- [x] Created a assets/payload_builder.py
- [x] Will show the current user's login name in C2 command prompt

rly0nheart committed:
Corrected typo in requirements.txt

rly0nheart committed:
Update c2c.py

rly0nheart committed:
Create payload_builder.py

rly0nheart committed:
Create banner.py

rly0nheart committed:
Delete payload.py

rly0nheart committed:
Update c2c.py

grav3m1nd-byte reviewed (commented):
I definitely like the changes. Not my decision to make of course as this is not my project, but definitely like it. I wanna try and test it, but it might take me a few days.

rly0nheart commented:
> 

Cheers! Let's wait to hear from the author

rly0nheart committed:
Create helper.py

rly0nheart committed:
Update c2c.py

1. Moved all the help menus to assets/helper.py
2. Help menus will be displayed in tables

rly0nheart committed:
Update helper.py

rly0nheart committed:
Update helper.py

rly0nheart committed:
Update README.md

rly0nheart committed:
Update banner.py

rly0nheart committed:
Update README.md

rly0nheart committed:
Update README.md

rly0nheart committed:
Update requirements.txt

rly0nheart committed:
Update README.md

rly0nheart committed:
Update helper.py

rly0nheart committed:
Update c2c.py

rly0nheart committed:
Update c2c.py

rly0nheart committed:
Update c2c.py

rly0nheart committed:
Merge branch 'main' into main
